app/views/user_mailer/reset_link_non_existent_account.html.erb

```diff
@@ -0,0 +1,14 @@
+<div>
+  <p>Hi there,</p>
+
+  <p>You have requested to reset password.</p>
+  
+  <p>But there is no SSID account registered with this email address.</p>
+
+  <p>You might want to take a look at <%= link_to "our website", "https://ssid.comp.nus.edu.sg/cover" %> and request for an account.</p>
```

Comment:
Use constants instead strings?  "ssid.comp.nus.edu.sg" => SSID_Host or similar?

Reply:
Yes, the hard-coded url was removed. Host will be different depending on the environment. It is made configurable.